relay-metrics/src/protocol.rs

```diff
@@ -213,26 +173,70 @@ impl fmt::Display for MetricNamespace {
     }
 }
 
-/// A metric name parsed as MRI, a naming scheme which includes most of the metric's bucket key
-/// (excl. timestamp and tags).
+/// A unique identifier for metrics including typing and namespacing.
+///
+/// MRIs have the format `<type>:<namespace>/<name>[@<unit>]`. The unit is optional and defaults to
+/// [`MetricUnit::None`].
+///
+/// # Statsd Format
+///
+/// In the statsd submission payload, MRIs are sent in a more relaxed format:
+/// `[<namespace/]name[@unit]`. The differences to the internal MRI format are:
```

Comment:
to match the format above:
```suggestion
/// `[<namespace>/]<name>[@<unit>]`. The differences to the internal MRI format are:
```